lib/pf/iptables.pm

```diff
@@ -140,6 +141,17 @@ sub iptables_generate {
     # per interface-type pointers to pre-defined chains
     ($tags{'filter_if_src_to_chain'},$tags{'filter_forward'}) = $self->generate_filter_if_src_to_chain();
 
+    $tags{'netdata'} = "-A input-management-if --protocol tcp --match tcp -s 127.0.0.1 --dport 19999 --jump ACCEPT\n";
+
+    if ($cluster_enabled) {
+        push my @mgmt_backend, map { $_->{management_ip} } pf::cluster::config_enabled_servers();
+
+        foreach my $mgmt_back (uniq(@mgmt_backend)) {
+            $tags{'netdata'} .= "-A input-management-if --protocol tcp --match tcp -s $mgmt_back --dport 19999 --jump ACCEPT\n";
+        }
+    }
+    $tags{'netdata'} .= "-A input-management-if --protocol tcp --match tcp --dport 19999 --jump DROP\n";
```

Comment:
Here the idea is to drop everything that is not related to a management IP or localhost, correct?

Reply:
Yes, since we reverse proxy the netdata interface, only ip addresses from the management and localhost needs to have access.